packages/flutter_tools/lib/src/commands/build_windows.dart

```diff
@@ -19,11 +20,22 @@ import 'build.dart';
 /// A command to build a windows desktop target through a build shell script.
 class BuildWindowsCommand extends BuildSubCommand {
   BuildWindowsCommand({
+    required OperatingSystemUtils operatingSystemUtils,
     bool verboseHelp = false,
-  }) : super(verboseHelp: verboseHelp) {
+  }) : _operatingSystemUtils = operatingSystemUtils,
+       super(verboseHelp: verboseHelp) {
     addCommonDesktopBuildOptions(verboseHelp: verboseHelp);
+    // As long as arm64 does not have artifacts available, stick to x64.
```

Comment:
Please add a TODO here as well since we'll want to update this later

```suggestion
    // As long as arm64 does not have artifacts available, stick to x64.
    // TODO: https://github.com/flutter/flutter/issues/62597
```